music21/metadata/testMetadata.py

```diff
@@ -452,21 +536,83 @@ def testUniqueNameAccess(self):
         self.checkUniqueNamedItem('dedicatedTo', 'humdrum:ODE')
         self.checkUniqueNamedItem('commissionedBy', 'humdrum:OCO')
         self.checkUniqueNamedItem('countryOfComposition', 'humdrum:OCY')
-
         self.checkUniqueNamedItem('localeOfComposition', 'humdrum:OPC')
         self.checkUniqueNamedItem('groupTitle', 'humdrum:GTL')
         self.checkUniqueNamedItem('associatedWork', 'humdrum:GAW')
         self.checkUniqueNamedItem('collectionDesignation', 'humdrum:GCO')
-        self.checkUniqueNamedItem('attributedComposer', 'humdrum:COA',
-            valueType=metadata.Contributor)
-        self.checkUniqueNamedItem('suspectedComposer', 'humdrum:COS',
-            valueType=metadata.Contributor)
-        self.checkUniqueNamedItem('composerAlias', 'humdrum:COL',
-            valueType=metadata.Contributor)
-        self.checkUniqueNamedItem('composerCorporate', 'humdrum:COC',
-            valueType=metadata.Contributor)
-        self.checkUniqueNamedItem('orchestrator', 'humdrum:LOR',
-            valueType=metadata.Contributor)
+        self.checkUniqueNamedItem(
+            'attributedComposer',
+            'humdrum:COA',
+            valueType=metadata.Contributor
+        )
+        self.checkUniqueNamedItem(
+            'suspectedComposer',
+            'humdrum:COS',
+            valueType=metadata.Contributor
+        )
+        self.checkUniqueNamedItem(
+            'composerAlias',
+            'humdrum:COL',
+            valueType=metadata.Contributor
+        )
+        self.checkUniqueNamedItem(
+            'composerCorporate',
+            'humdrum:COC',
+            valueType=metadata.Contributor
+        )
+        self.checkUniqueNamedItem(
+            'orchestrator',
+            'humdrum:LOR',
+            valueType=metadata.Contributor
+        )
+        self.checkUniqueNamedItem(
+            'firstPublisher',
+            'humdrum:PPR',
+            valueType=metadata.Contributor
+        )
+        self.checkUniqueNamedItem(
+            'dateFirstPublished',
+            'humdrum:PDT',
+            valueType=metadata.DateSingle
+        )
+        self.checkUniqueNamedItem('publicationTitle', 'humdrum:PTL')
+        self.checkUniqueNamedItem('placeFirstPublished', 'humdrum:PPP')
+        self.checkUniqueNamedItem('publishersCatalogNumber', 'humdrum:PC#')
```

Comment:
ugh, I can't believe that humdrum uses '#' as part of the three-letter code.  Well, nothing we can do.  This makes parsing a text file with them harder.